### ISSUE TITLE
Document relevant WCAG2.0 standards for each test

### ISSUE THREAD

**Issue by ckundo** [GoogleChrome/accessibility-developer-tools#18]
Product owners, especially those with a Section 508 compliance criteria, need to have a mapping between the rules and W3C standards. README might be the right place to start.

**Comment by ckundo** [comment]
It might be useful to reference this list of test cases and related criteria by the auto-WCAG group at W3C. There are at least a few that map directly to ADT rules. http://www.w3.org/community/auto-wcag/wiki/Test_overview

**Comment by thecristen** [comment]
can't +1 this enough

**Comment by JoeMurray** [comment]
Without knowing if a test is passing or failing WCAG 1.0, 2.0, A or AA or AAA the tool seems problematic. Even just indicating what standard it is intending to implement for checks would be a good start. Enhancement would be to allow the user to select which standard for a run.
